### PR TITLE
Update example for static files

### LINKED ISSUE #796: Update example for static files
this section - https://render-engine.readthedocs.io/en/latest/theme_management/#handling-static-files.
```python
from render_engine import Site

class MySite(Site):
     static_path = "my_static"
 # You can also set this after you've initialized the class
 # app = MySite()
 # app.static_path = "my_static"
```

to

```python
from render_engine import Site

app = MySite()
app.static_paths.add = "my_static"
```

### PR DESCRIPTION
**Pull request by OsamaRab3**
fix #796
#### Type of Issue

- [ ] :bug: (bug)
- [x] :book: (Documentation)
- [ ] :arrow_up: (Update)
- [ ] :dizzy: (New Feature)
- [ ] :radioactive: (Deprecation)
- [ ] :no_entry_sign: (Removal)
- [ ] :hammer_and_wrench: (Refactor)

#### Changes have been Documented (If Applicable)

- [x] YES - Changes have been documented

#### Changes have been Tested

- [x] YES - Changes have been tested

### CODE REVIEW

File: docs/docs/theme_management.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ Unlike templates you can have more than one static_path you wish to include. You
 ```python
 from render_engine import Site
 
+class MySite(Site):
+    static_path = "my_static"
+
```
Comment (kjaymiller) [inline docs/docs/theme_management.md:44]:
```suggestion
    ```
Another option is to apply the static path after initialization as the paths are not merged until `Site.render()` is called.
```

Comment (kjaymiller) [reply docs/docs/theme_management.md:44]: You will likely need to make this change in your code as I can't escape the code blocks in the suggestion.

**Review by kjaymiller — CHANGES_REQUESTED** [review]

**Comment by OsamaRab3** [comment]
Is the following code snippet what you're expecting?
--- 

```python
from render_engine import Site

class MySite(Site):
    static_path = "my_static"
```
Another option is to apply the static path after initialization as the paths are not merged until `Site.render()` is called.

```python

app = MySite()
app.static_paths.add = "my_static"

```

**Comment by kjaymiller** [comment]
The failing action is related to another issue and this change does not introduce a new problem.

**Comment by OsamaRab3** [comment]
> The failing action is related to another issue and this change does not introduce a new problem.

I tried running ruff check . locally, and it's working fine on my local machine

image

**Comment by kjaymiller** [comment]
it's not ruff it's markdownlint and it's based on the all contributors tag.

**Pull request merged**